integration-tests/http/__tests__/auth/admin/auth.spec.ts

```diff
@@ -198,13 +198,11 @@ medusaIntegrationTestRunner({
           },
         })
 
-        const response = await api.post(
-          `/auth/user/emailpass/update?token=${result}`,
-          {
-            email: "test@medusa-commerce.com",
-            password: "new_password",
-          }
-        )
+        const response = await api.post(`/auth/user/emailpass/update`, {
+          token: result,
```

Comment:
Should we pass the token in a header instead?

Reply:
That's also an option yeah, we can pass it as an authorization header, so it's always redacted

Reply:
I think if there are no issues with that it would be more consistent with the rest of our authentication setup, but I leave it up to you to decide